src/Microsoft.Data.Sqlite/Interop/VersionedMethods.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Data.Sqlite.Interop
+{
+	internal class VersionedMethods
```

Comment:
Unified all version-switched calls into one place. @bricelam...name suggestion for this class?